src/Effects.js

```diff
@@ -1445,6 +1445,34 @@ module.exports = class Effects {
         }
       }
     };
+
+    this.music_wave = {
+      inc: 360/15,
+      heightFactor: 1,
+      heightDivider: 200,
+      yLoc: 300,
+      lineWidth: p5.width/80,
+      angle: 0,
```

Comment:
One last clean-up! It doesn't look like we use this angle variable but instead use the angle variable on line 1459. If that's correct, let's get rid of this variable.

Reply:
Sorry about that. Seems like I left it out when changed the type of variable.